schemas/cloudToDevice/cell_position/cell-position.json

```diff
@@ -36,7 +36,8 @@
     "additionalProperties":false,
     "required":[
        "appId",
-       "messageType"
+       "messageType",
+       "data"
```

Comment:
https://github.com/nRFCloud/application-protocols/blob/b7fab0e679d76e523b1f36d8e843c6f5183217df/schemas/cloudToDevice/cell_position/cell-position-err-example.json

data is not required in the case of an error.